teps/0114-custom-tasks-beta.md

```diff
@@ -208,6 +209,19 @@ Remove guarding of `Custom Tasks` behind `enable-custom-tasks` and `enable-api-f
 When [TEP-0096: Pipelines V1 API][tep-0096] is implemented to add V1 API, `Custom Tasks` will be gated behind feature
 gate `enable-api-fields` being set to `"beta"` - this is out of scope for this TEP  (in scope for TEP-0096).
 
+##### Cancellation
+
+Implementing cancellation is now `required` instead of `optional`. If the Custom Task implementor does not support cancellation via `.spec.status`, `Pipeline` timeouts and `PipelineRun` cancellation will not work properly.
+
+Pipeline Controller sets the `spec.Status` and `spec.StatusMessage` to signal `CustomRuns` about the `Cancellation`, while `CustomRun` controller **MUST** update its `status.conditions` as following once noticed the change on `spec.Status`.
+
+| Pipeline Signal| `spec.Status` | `spec.StatusMessage` | `status.Conditions` | 
+|:--- |:--- |:--- |:---|
+|Parent `PipelineRun` is cancelled | `RunCancelled` | `CustomRun cancelled as the PipelineRun it belongs to has been cancelled. `| `Type: Succeeded`<br>`Status:False`<br>`Reason:CustomRunCancelled` |
+|`Pipeline` timeouts | `RunCancelled` | `CustomRun cancelled as the PipelineRun it belongs to has timed out.`| `Type: Succeeded`<br>`Status:False`<br>`Reason:CustomRunCancelled`|
+
+Note: Custom Task implementors **MUST NOT** update the value of `spec.StatusMessage` and `spec.Status`, because Pipeline Controller depends the two fields.
```

Comment:
NIT: depends on the two fields - missing on

Reply:
Thanks! Done~